sandbox/anvi-script-reformat-fasta

```diff
@@ -160,10 +177,15 @@ if __name__ == '__main__':
                         help="Maximum fraction of gaps in a sequence (any sequence with \
                               more gaps will be removed from the output FASTA file). The \
                               default is %(default)f.")
+    parser.add_argument('-M','--max-gaps', type=float, default=1000000,
+                        help="Maximum amount of gaps allowed per sequence in the alignment.")
```

Comment:
How about adding something like this: "Don't know what to pick? Use the --export-gap-counts-table to decide an appropriate number"

Reply:
Gread idea!